hookshot-dashboard.json

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
If I had to leave the tiniest of nits, could this go into a `contrib/` directory which is what Synapse do I think. Can you then link the file in the metrics documentation so people can find it?